main/reference/ertp-api/amount-math.md

```diff
@@ -213,11 +206,11 @@ const combinedList = AmountMath.add(listAmountA, listAmountB);
 
 Returns a new **Amount** that is the *leftAmount* minus the *rightAmount* (i.e., 
 everything in the *leftAmount* that is not in the *rightAmount*). If *leftAmount* 
-doesn't include *rightAmount* (i.e., subtraction results in a negative), this method throws an 
-error. Because *leftAmount* must include *rightAmount*, this is **not** 
+doesn't include *rightAmount* (e.g., subtraction results in a negative), an error is thrown.
+Because *leftAmount* must include *rightAmount*, this is **not** 
 equivalent to set subtraction.
```

Comment:
Why e.g.? If the leftAmount doesn't include rightAmount, won't that always means that subtracgtion will result in a negative?

Reply:
No, because Amounts are not limited to numbers. For example, subtracting the [set](https://docs.agoric.com/reference/ertp-api/ertp-data-types.html#assetkind) {1, 2} from the set {0, 1} does not result in a negative but still throws an error.

Reply:
K.